gymnasium/envs/mujoco/mujoco_rendering.py

```diff
@@ -695,16 +740,17 @@ def _get_viewer(self, render_mode: str):
         self.viewer = self._viewers.get(render_mode)
         if self.viewer is None:
             if render_mode == "human":
-                self.viewer = WindowViewer(
-                    self.model, self.data, self.width, self.height
-                )
-
+                viewer_class = WindowViewer
```

Comment:
Why change the initialization method of `self.viewer`

Reply:
now that both viewers have the same init arguments, I figured it would be cleaner to have them initialized in one place. a personal preference of mine, but not necessary

Reply:
fixed in 7f9b9ee